docs/video/ffmpeg/upscaling.md

```diff
@@ -50,12 +50,13 @@ Here are two videos, one was left as is, the other one stretched to 4K, watch it
     
 === "1080p stretched to 4K"
 
-    <iframe width="688" height="387" src="https://www.youtube.com/embed/-FE9s_acdYw?color=white" frameborder=0 allowfullscreen></iframe>
+    <iframe width="688" height="387" src="https://www.youtube.com/embed/-FE9s_acdYw" frameborder=0 allowfullscreen></iframe>
```

Comment:
please revert these white looks cool

Reply:
they looked the same idk

Reply:
> they looked the same idk

![image](https://github.com/couleur-tweak-tips/CTT/assets/82747632/10bf026c-3ed5-41e1-a14e-988d6353b1f0)
is that not white for you?